src/FsLibLog/FsLibLog.fs

```diff
@@ -1211,4 +1196,21 @@ module LogProvider =
     let getCurrentLogger () =
         let stackFrame = StackFrame(2, false)
         getLoggerByType (stackFrame.GetMethod().DeclaringType)
+
+
+type LogProvider =
+    /// <summary>
+    /// Creates a logger based on `Reflection.MethodBase.GetCurrentMethod().FullName` and `CallerMemberName`.  This is only useful for calls within functions. This does work on inlined functions.
+    /// </summary>
+    /// <param name="memberName"></param>
+    /// <returns></returns>
+    static member inline getLoggerByFunc([<System.Runtime.CompilerServices.CallerMemberName>] ?memberName: string) =
+        let mi = System.Reflection.MethodBase.GetCurrentMethod()
+        // When we're in a CE we get something like `WebBackend.App+thingsToCall2@130`.
+        // CallerMemberName gets us the function that actually called it
+        // Splitting off + seems like the best option to get the Fully Qualified Path
+        let location = mi.DeclaringType.FullName.Split('+') |> Seq.tryHead |> Option.defaultValue ""
```

Comment:
This splitting code is definitely suspect/smelly but I don't see any obvious alternatives. Definitely open to options.